eth/catalyst/api.go

```diff
@@ -160,6 +161,17 @@ type ConsensusAPI struct {
 	newPayloadLock sync.Mutex // Lock for the NewPayload method
 }
 
+var (
+	// Number of blobs requested via getBlobsV2
+	getBlobsV2BlobsRequestedTotal = metrics.NewRegisteredCounter("get_blobs_requests_blobs_total", nil)
+	// Number of blobs requested via getBlobsV2 that are present in the blobpool
+	getBlobsV2BlobsInPoolTotal = metrics.NewRegisteredCounter("get_blobs_requests_blobs_in_blobpool_total", nil)
+	// Number of times getBlobsV2 responded with “hit”
+	getBlobsV2RequestHit = metrics.NewRegisteredCounter("get_blobs_requests_success_total", nil)
+	// Number of times getBlobsV2 responded with “miss”
+	getBlobsV2RequestMiss = metrics.NewRegisteredCounter("get_blobs_requests_failure_total", nil)
```

Comment:
```suggestion
	getBlobsRequestedCounter = metrics.NewRegisteredCounter("engine/getblobs/requested", nil)
	// Number of blobs requested via getBlobsV2 that are present in the blobpool
	getBlobsAvailableCounter = metrics.NewRegisteredCounter("engine/getblobs/available", nil)
	// Number of times getBlobsV2 responded with “hit”
	getBlobsV2RequestHit = metrics.NewRegisteredCounter("engine/getblobs/hit, nil)
	// Number of times getBlobsV2 responded with “miss”
	getBlobsV2RequestMiss = metrics.NewRegisteredCounter("engine/getblobs/miss", nil)
```

Reply:
In the prometheus exporter, we replace `/` by `_`. So our names are aligned with the Prometheus convention. We can discuss the names, but I feel that they should at least contain the `engine` prefix. We have a lot of metrics and they are all categorized nicely. Don't want to break this.